todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
+    created_at = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField()
+    is_done = models.BooleanField(default=False)
+    tags = models.ManyToManyField(to=Tag, related_name="tasks")
+
+    class Meta:
+        ordering = ["created_at", "is_done"]
```

Comment:
Let's order first by `is_done` and then by `created_at`

```suggestion
        ordering = ["is_done", "created_at"]
```